src/metabase/util.cljc

```diff
@@ -1118,3 +1120,27 @@
   "Return first item from Reducible"
   [reducible]
   (reduce (fn [_ fst] (reduced fst)) nil reducible))
+
+(defn rconcat
```

Comment:
Was surprised not to find this in neither `clojure.core` nor `clojure.core.reducers`, maybe I missed it somehow.

Reply:
Ah, there is `clojure.core.reducers/cat` *but* that only works for concrete collections.